### PR TITLE
support feature id list for expression search

### PR DESCRIPTION
**Pull request by saupchurch**
contains two fixes for expression search:
1) adds support for feature_ids
2) removes erroneous expression id requirement in rnaquantification back end

### CODE REVIEW

File: ga4gh/cli.py [hunk 1]
```diff
@@ -797,11 +797,14 @@ class SearchExpressionLevelsRunner(AbstractSearchRunner):
     def __init__(self, args):
         super(SearchExpressionLevelsRunner, self).__init__(args)
         self._rnaQuantificationId = args.rnaQuantificationId
+        if len(args.feature_ids) > 0:
```
Comment (dcolligan) [inline ga4gh/cli.py]: what happens if `len(args.feature_ids) <= 0`?  will it error in `run` because `self._featureIds` does not exist?

**Comment by dcolligan** [comment]
looks good aside from my comment

**Comment by david4096** [comment]
+1 merge at will

File: ga4gh/cli.py (hunk 1 above)
Comment (saupchurch) [inline ga4gh/cli.py]: good catch will fix

**Comment by saupchurch** [comment]
fixed, squashed.  Can I get a renewal of the +1? @david4096 @dcolligan

File: ga4gh/cli.py (hunk 1 above)
Comment (dcolligan) [inline ga4gh/cli.py]: nit: we usually do this like 
```
self._feature_ids = []
if len(args.feature_ids) > 0:
    self._feature_ids = args.feature_ids.split(",")
```
Cuts down on one line and ensures something is assigned to the attribute, even if the things in the if/else clauses change

**Comment by dcolligan** [comment]
+1

**Pull request merged**